test/extended/cmd.sh

```diff
@@ -11,9 +11,9 @@ set -o pipefail
 
 OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/hack/cmd_util.sh"
```

Comment:
I didn't reconcile merge conflicts, instead I copy-pasted mine in and made changes to approach yours. I wasn't focused on the conflicts in this area.